src/style.css

```diff
@@ -0,0 +1,110 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: normal;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-style: bold;
+  src: url(fonts/Avenir.ttc);
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  padding: 0;
+  margin-top: 0;
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.big_form {
+  width: 100%;
+  height: 70px;
+  border-color: #e1e7ed;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding-left: 62px;
+}
+
+.big_form::placeholder {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  color: #3d4e61;
+  font-weight: 400;
+}
+
+.small_form::placeholder {
+  font-family: "Avenir", sans-serif;
+  color: #3d4e61;
+  font-size: 14px;
+  font-weight: 400;
+}
+
+.small_form {
+  width: 100%;
+  height: 42px;
+  border-color: #e1e7ed;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding-left: 33px;
+  padding-bottom: 1px;
+}
+
+.big_form:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.small_form:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.big_form:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-family: "Avenir", sans-serif;
+  font-weight: bold;
+}
+
+.small_form:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-family: "Avenir", sans-serif;
+  font-weight: bold;
+}
```

Comment:
check out the point 2 in the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)